src/services/codefixes/helpers.ts

```diff
@@ -921,7 +918,6 @@ function replaceFirstIdentifierOfEntityName(name: EntityName, newIdentifier: Ide
     return factory.createQualifiedName(replaceFirstIdentifierOfEntityName(name.left, newIdentifier), name.right);
 }
 
-/** @internal */
 export function importSymbols(importAdder: ImportAdder, symbols: readonly Symbol[]) {

```

Comment:
```suggestion
/** @internal */
export function importSymbols(importAdder: ImportAdder, symbols: readonly Symbol[]) {
```

This looks like a mistake in the rebase.

Reply:
We removed the `export` keyword in this PR, but then it got imported in main again.